core/app/models/spree/zone.rb

```diff
@@ -70,6 +70,10 @@ def self.match(address)
       matches.first
     end
 
+    def self.default_checkout_zone
+      find_by(name: Spree::Config[:checkout_zone]) || find_by(name: Spree.t(:default_zone))
```

Comment:
```suggestion
      find_by(name: Spree::Config[:checkout_zone])
```

let not set to any zone if it isn't set at all